lib/adal.js

```diff
@@ -1246,12 +1394,20 @@ var AuthenticationContext = (function () {
         return copy;
     };
 
+    /**
+     * Adds the library version and returns it.
+     * @ignore
+     */
     AuthenticationContext.prototype._addLibMetadata = function () {
         // x-client-SKU
         // x-client-Ver
         return '&x-client-SKU=Js&x-client-Ver=' + this._libVersion();
     };
 
+    /**
+     * Checks the Logging Level and constructs the Log message.
+     * @ignore
```

Comment:
public method. Besides constructing the log message, it logs the message also. In the description, we should specify that users should implement/override this method to turn on logging.